common/lib/xmodule/xmodule/js/spec/video/video_player_spec.js

```diff
@@ -380,22 +380,21 @@ function(VideoPlayer, HLS) {
             });
 
             describe('when the video is playing', function() {
-                beforeEach(function(done) {
+                beforeEach(function() {
                     state.videoPlayer.play();
+                });
 
+                it('call runTimer in seekTo on player', function(done) {
                     jasmine.waitUntil(function() {
```

Comment:
@Qubad786 Other two tests can be tested without video playing because we can do `seek` and `slide` independently whether the video is playing or not. In first test we need `jasmine.waitUntil` to ensure that video is playing before we do `expects` because we are explicitly verifying that `stopTimer` is called. And in our context `stopTimer` is called only when video [is playing](https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/js/src/video/03_video_player.js#L457).